### ISSUE TITLE
Adds unittests to the project

### ISSUE THREAD

**Issue by karlcow** [webcompat/issue_parser#18]
This was a one-off script, but if this becomes something more used/usable we probably need to pour a bit of testing into it.

* [ ] create a tests directory.
* [ ] create tests for the main functions.
* [ ] optimize the functions if not testable as-is.

**Comment by skulltech** [comment]
Which framework should we use for testing? I think [`pytest`](https://pytest.org/) would be good.

**Comment by karlcow** [comment]
Yeah pytest is fun, but on webcompat.com, we use unittest from python directly and nose to run the tests. For simplicity I would prefer we stick to this.

**Comment by skulltech** [comment]
Okay yeah, understandable.
